pkg/reconciler/apiserversource/apiserversource.go

```diff
@@ -55,15 +53,6 @@ const (
 	component = "apiserversource"
 )
 
-var apiServerEventTypes = []string{
```

Comment:
I thought this still needs to know what it will make to put these in the CRD status? from this change: https://github.com/knative/pkg/pull/1165

Reply:
yeah, the previous code registered all of the types. So I kept that. But you say we shouldn't be registering all the types, and it should be based on what we configured on the Source CO?

Reply:
they are just moved... I was still reading 🐎